cirq/study/trial_result.py

```diff
@@ -108,7 +108,7 @@ def data(self) -> pd.DataFrame:
                 converted_dict[key] = [
                     value.big_endian_bits_to_int(m_vals) for m_vals in val
                 ]
-            self._data = pd.DataFrame(converted_dict)
+            self._data = pd.DataFrame(converted_dict, dtype=np.int64)
```

Comment:
Could there be any problems in the future from hard coding this to int64?

Reply:
See other comment.